test/integration/homepage_popular_links_test.rb

```diff
@@ -51,6 +51,48 @@ class HomepagePopularLinksTest < JavascriptIntegrationTest
     end
   end
 
+  context "#confirm_destroy" do
+    setup do
+      click_button("Create new edition")
+    end
+
+    should "show the 'Delete draft' confirmation page with the option to actually delete" do
+      click_link("Delete draft")
+
+      assert page.has_text?("Delete draft")
+      assert page.has_text?("Are you sure you want to delete this draft?")
+      assert page.has_text?("Cancel")
+    end
+
+    should "revert to show page if delete is cancelled" do
```

Comment:
probably we can use the word 'navigate' instead of revert.
also the test is performing actions so we can rename the test to something like
"navigate to show page when 'Cancel' button is clicked"